patches/expo-image-manipulator+13.0.5+002+fix-type-exports.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/expo-image-manipulator/src/index.ts b/node_modules/expo-image-manipulator/src/index.ts
```

Comment:
Can you please make sure all the new patches added here are tracked? I see you have added them to the list in the PR, but just reminding here if its up to date?

Reply:
I will go through all of them when bumping to RN 0.77 ok?